utilities/mobile_server_pool.py

```diff
@@ -94,6 +96,10 @@ def check_vm_alive(server):
 
 
 def reserve_node(doc_id, job_name, counter=0):
+    return True
```

Comment:
is this function no longer valid? if this is the case, it's better to remove it, or at least print out some message to let user know. otherwise, it returns true but does nothing, it will confuse user and user may have to spend a lot of time to find out what's going on.